dags/collisions_asc_mv.py

```diff
@@ -0,0 +1,69 @@
+#This script should run after the MOVE dag dumps data into collisions_replicator."ACC"
+#Also I heavily cribbed this off of refresh_wys_monthly.py...
+
+# Operators; we need this to operate!
+from airflow import DAG
+from datetime import datetime, timedelta
+from airflow.operators.postgres_operator import PostgresOperator 
+from airflow.hooks.base_hook import BaseHook
+from airflow.contrib.operators.slack_webhook_operator import SlackWebhookOperator
+SLACK_CONN_ID = 'slack'
+
+#This script does things with those operators:
+#1) uses ACC to update acc_safe_copy
+#2) updates the mat view called collisions_no, which generates ids needed to...
+#3) update the events and involved mat views 
+#4) throws a sassy slack alert message when it fails
+
+def task_fail_sassy_slack_alert(context):
+    slack_webhook_token = BaseHook.get_connection(SLACK_CONN_ID).password
+    task_msg = 'The {task} in updating collisions failed, <@U02NSCSKFEU> go fix it meow :meow_tableflip: '.format(
+            task=context.get('task_instance').task_id,) #K just gotta point out that the last 3 letters in my slack id spell FIRE in French!!!   
+        
+    slack_msg = task_msg + """(<{log_url}|log>)""".format(
+            log_url=context.get('task_instance').log_url,)
+    failed_alert = SlackWebhookOperator(
+        task_id='slack_test',
+        http_conn_id='slack',
+        webhook_token=slack_webhook_token,
+        message=slack_msg,
+        username='airflow'
+        )
+    return failed_alert.execute(context=context)
+
+default_args = {'owner':'scannon',
+                'depends_on_past':False,
+                'start_date': datetime(2022, 5, 26), #start this Thursday, why not?
+                'email': ['sarah.cannon@toronto.ca'],
+                'email_on_failure': False,
+                 'email_on_success': False,
+                 'retries': 0,
+                 'retry_delay': timedelta(minutes=5),
+                 'on_failure_callback': task_fail_sassy_slack_alert
+                }
+
+
+with DAG('BDITTO_ACC_REPLICATOR', # going waaaaaayyyyy out on a limb of the magical assumption tree here....
```

Comment:
I appreciate I haven't documented this, and alllll the other dags are bad examples of this, but I would like dags to be named in the format: `dataset_action` (also all lower case plz), so a good candidate here would be `collisions_replicator_transfer` or some such.

Reply:
Wait did you mean that the dag name 'BDITTO_ACC_REPLICATOR' should be changed? Because I changed the name of the .py file to collisions_replicatory_transfer.py and maybe that was a misunderstanding?

Reply:
I think Raph meant changing the DAG name. This name is what we see in Airflow UI. 
Also, the file name now is more meaningful